static/openapi/prerendering-spec.json

```diff
@@ -3060,1025 +3384,1091 @@
+            "in": "query",
+            "name": "limit",
+            "required": false,
+            "description": "Pagination limit."
+          },
           {
             "schema": { "type": "string", "minLength": 1 },
             "in": "path",
             "name": "origin",
             "required": true,
-            "description": "Origin hostname"
+            "description": "Origin hostname."
           },
           {
             "schema": { "type": "string" },
             "in": "header",
             "name": "x-photoniq-customerid",
             "required": true,
-            "description": "Customer unique identifier"
+            "description": "Customer unique identifier."
           }
         ],
         "responses": {
           "200": {
-            "description": "Successful response",
+            "description": "Successful response.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Successful response",
+                  "description": "Successful response.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Success status"
+                      "description": "Success status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Success message"
+                      "description": "Success message."
                     },
                     "data": {
-                      "type": "object",
-                      "properties": {
-                        "_key": {
-                          "type": "string",
-                          "description": "Prefetch configuration unique identifier."
-                        }
-                      },
-                      "required": ["_key"]
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "url": {
+                            "type": "string",
+                            "format": "uri",
+                            "description": "Origin URL."
+                          },
+                          "isMobile": {
+                            "type": "boolean",
+                            "description": "Whether the URL is rendered for mobile devices or not."
+                          },
+                          "status": {
+                            "type": "string",
+                            "enum": [
+                              "queued",
+                              "retrying",
+                              "completed",
+                              "failed"
+                            ],
+                            "description": "URL status."
+                          },
+                          "createdAt": {
```

Comment:
Can you standardize on the message for **cratedAt**

Somewhere else you had this:  _Date when the prefetch configuration was created._
And here you have : _Tmestamp of when the prefetch URL was created._

Standardize on one and use it the samewhy throughout the document.

Reply:
Yeah, we discussed this a few times. I'll bring up this topic again tomorrow.
The prefetch URL uses a timestamp because the collection has a TTL index. But in this case, we could convert it to a Date string before returning the data.